app/src/features/apiClient/screens/environment/components/environmentView/EnvironmentView.tsx

```diff
@@ -87,6 +89,10 @@ export const EnvironmentView = () => {
     return setVariables(persistedEnvId, pendingVariables)
       .then(() => {
         toast.success("Variables updated successfully");
+        trackVariablesUpdated({
+          type: isGlobalEnvironment(envId) ? "global_variables" : "environment_variable",
+          updated_variable_count: getUpdatedVariableCount(variables, pendingVariables),
+        });
```

Comment:
Refer to comment in `app/src/features/apiClient/screens/apiClient/components/clientView/components/Collection/components/CollectionsVariablesView/CollectionsVariablesView.tsx`